bench/src/main/scala/benchmarks/BinaryFenBench.scala

```diff
@@ -47,6 +47,7 @@ class BinaryFenBench:
       val game = games(i)
       Blackhole.consumeCPU(Work)
       bh.consume(BinaryFen.write(game))
+      i += 1
```

Comment:
:sweat_smile:

Reply:
haha, I don't know how did I run it last time.